src/crate_details.rs

```diff
@@ -146,6 +147,55 @@ impl CrateDetails {
         Ok(())
     }
 
+    // strip the features that point to dev deps so we don't get errors when we strip the dev deps
+    fn strip_dev_features(&self, toml: &mut Document) {
+        let feature_names = toml
+            .get("features")
+            .and_then(|item| item.as_table())
+            .map(|item| {
+                item.iter()
+                    .map(|entry| entry.0.to_string())
+                    .collect::<HashSet<_>>()
+            });
+
+        let feature_names = if let Some(feature_names) = feature_names {
+            feature_names
+        } else {
+            return;
+        };
```

Comment:
nit:
```suggestion
        let Some(feature_names) = feature_names else {
            return;
        };
```

Reply:
Neat. Not really played with let else before. Seems useful.